MCLEN/iocBoot/iocMCLEN-IOC-01/st-motor-init.cmd

```diff
@@ -0,0 +1,28 @@
+## Initialises values at the motor level. Experience suggests the IOC doesn't do this automatically
+
+asynOctetConnect("MKINIT","$(ASERIAL)")
+
+## Some things that are helpful in development mode. Resetting, initialising. Probably don't want to do it in production
+#asynOctetWrite("MKINIT","$(MN)RS")
+#asynOctetWrite("MKINIT","$(MN)IN")
+
+## Check if open loop mode has been requested
+stringiftest("CMOPEN", "$(MODE$(MN)=)",4,"OPEN")
+
+## Initialise control mode. Defaults to CM14, closed
+$(IFCMOPEN) asynOctetWrite("MKINIT","$(MN)CM11")
+$(IFNOTCMOPEN) asynOctetWrite("MKINIT","$(MN)CM14")
+
+## Initialise the encoder ratio using the exact string of ERES$
+asynOctetWrite("MKINIT","$(MN)ER$(ERES$(MN)=400/4096)")
+
+#### I'd rather these not go into production yet, it might change everyone's motor velocities/acceleration
+### Initialise values. Need to convert to motor units
```

Comment:
If this code isn't production-ready, should it be removed completely rather than commented out? It will always be in git...